egs2/aishell/asr1/local/download_and_untar.sh

```diff
@@ -0,0 +1,105 @@
+#!/bin/bash
```

Comment:
Is this file same as the original? If so, use symbolic link instead of copying.

Reply:
I made some minor changes to the original one to pass `shellcheck`.
> For example,
>   `if ! which wget >/dev/null; then` →  `if ! command -v wget >/dev/null; then`
>    `cd $data`   →  `cd $data || exit 1`

Reply:
o.k.

Reply:
This change seems to be common and we could modify the original `download_and_untar.sh` with the above modifications and use the symlink.